module-2/backend/week_one.md

```diff
@@ -5,30 +5,78 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  GET - get an existing page
+  POST - create a new page
+  PUT - update an existing page
+  DELETE - delete a page
+  
 2. What is Sinatra?
-4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
-7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
+  A Domain Specific language

```

Comment:
Yes, but I might've also included some detail about the fact that it's a DSL that we use to create dynamic web pages.